CMakeLists.txt

```diff
@@ -121,7 +121,7 @@ if (BUILD_WITH_COMPILER STREQUAL "HIP-NVCC" )
 
   else()
   # Define GPU targets
-  set( AMDGPU_TARGETS gfx803;gfx900;gfx906:xnack-;gfx908:xnack-;gfx90a:xnack-;gfx90a:xnack+;gfx1030;gfx1100;gfx1101;gfx1102 CACHE STRING
+  set( AMDGPU_TARGETS gfx803;gfx900;gfx906;gfx908;gfx90a;gfx940;gfx941;gfx942;gfx1030;gfx1100;gfx1101;gfx1102 CACHE STRING
```

Comment:
If there is no clients device code need to be compiled, we should just remove those